build.gradle

```diff
@@ -130,20 +135,73 @@ dependencies {
 
     // These dependencies are only needed for running tests
     testCompile group: 'junit', name: 'junit', version: '4.11'
-    testCompile group: 'org.mockito', name: 'mockito-all', version: '1.10.19'
+    testCompile group: 'org.mockito', name: 'mockito-core', version: '2.26.0'
+    testCompile group: 'org.mockito', name: 'mockito-inline', version: '2.26.0' // For final classes
 
-    // Don't upgrade either of these until https://github.com/powermock/powermock/issues/717 is resolved
-    testCompile 'org.powermock:powermock-module-junit4:1.6.4'
-    testCompile 'org.powermock:powermock-api-mockito:1.6.4'
+    testCompile 'org.powermock:powermock-module-junit4:2.0.4'
+    testCompile 'org.powermock:powermock-api-mockito2:2.0.2'
 }
 
 // Set the expected module Java level (can use a higher Java to run, but should not use features from a higher Java)
-sourceCompatibility = 1.8
-targetCompatibility = 1.8
+sourceCompatibility = 1.11
+targetCompatibility = 1.11
+
+javafx {
+    version = "11.0.2"
+    modules = [
+            'javafx.graphics',
+            'javafx.fxml',
+            'javafx.web']
+}
+
+jlink {
+    launcher {
+        name = 'launcher'
+    }
+
+    imageName = "TerasologyLauncher"
+
+    ['Linux64',
+    'Linux32',
+    'Windows64',
+    'Windows32',
+    'Mac'].each { name ->
+        targetPlatform(name) {
+            jdkHome = "jre/Launcher/$name/"
+        }
+    }
+}
+
+tasks.getByName("jlink").dependsOn "downloadJreAll" // For creating images for platforms.
+tasks.getByName("jlink").doLast {
+    ['Linux64',
+     'Linux32',
+     'Windows64',
+     'Windows32',
+     'Mac'].each { platform ->
+        copy {
+            from file("jre/Game/$platform")
```

Comment:
Is this to support for a different JRE for the game itself? With recent talk on Discord it looks like we don't need that? 
Probably need to add more modules to be bundled then: https://badass-jlink-plugin.beryx.org/releases/latest/#_customimage